instructions/FOSS_staging_list.pl

```diff
@@ -20,11 +20,9 @@
       "deploy_pkg_into" => "zimbra-zextras",
    },
    {
-      "dir"         => "zm-timezones",
-      "ant_targets" => ["publish-local"],
-      "stage_cmd"   => sub {
-         System("(cd .. && rsync -az --relative zm-timezones $CFG{BUILD_DIR}/)");
-      },
+      "dir"             => "zm-timezones",
+      "ant_targets"     => ["pkg"],
+      "deploy_pkg_into" => "zimbra-foss",
```

Comment:
Yes, with this commit, we are creating a different package out of zm-timezones, so that it can be potentially it be independently updated (out of the release cycle).

zimbra-foss is a logical name for the package repository that zimbra will have to host publically, so that users can be download them (via package management tools) from repository server. "deploy_pkg_into" is meant to specify a package repository name, where packages should get deployed when a build is created.

In my previous commit, I have disabled "deploy_pkg_into => zimbra-foss", that means the new package will simply get bundled into the tarball like others are, and we can continue our current tarball installations, until we are ready for repository deployments.